matrix_client/user.py

```diff
@@ -25,13 +25,24 @@ def __init__(self, api, user_id, displayname=None):
         self.displayname = displayname
         self.api = api
 
-    def get_display_name(self):
-        """ Get this users display name.
-            See also get_friendly_name()
+    def get_display_name(self, room=None):
+        """Get this user's display name.
+
+        See also get_friendly_name().
+
+        Args:
+            room (Room): Optional. If absent, return the global displayname for the user.
```

Comment:
Docstring needs to note that this is a room object rather than a room id.

Reply:
Does it? I see how someone reading the doc too fast would pass a `str` were it's very clearly written `Room`, but for me they'd deserve their `AttributeError`. Also, in case you didn't notice, Sphinx is smart enough to automatically add a link (clicking `Room` brings you to the room class), so there is really 0 ambiguity.  
Another concern is that for consistency we'd have to add those notes to every methods that take an attribute `room (Room)` or `user (User)`, which has a very low appeal to me...

Reply:
You're right. This was me forgetting about the sphinx convention that already tells you the type of an argument. No need to change anything.